bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/AbstractRetryingOperation.java

```diff
@@ -295,7 +295,8 @@ protected Long getNextBackoff() {
         return null;
       }
     } else {
-      return currentBackoff.getRetryDelay().toMillis();
+      return Math.min(currentBackoff.getRetryDelay().toMillis(),
+          currentBackoff.getRandomizedRetryDelay().toMillis());
```

Comment:
I'm a bit confused, shouldnt we just use `getRandomizedRetryDelay`?

Reply:
Yes, this makes more sense to me. I shall update it to `getRandomizedRetryDelay`